### ISSUE TITLE
gc.pyi is not read by PyRight

### ISSUE THREAD

**Issue by Josverl** [Josverl/micropython-stubber#270]
- [ ] gc
- [ ] sys
- [ ] io
- [ ] os

**Comment by Josverl** [comment]
https://github.com/Josverl/micropython-stubber/blob/dev/use_modules_done/snippets/common/validate_gc.py

```
pyright 1.1.270
C:\develop\MyPython\micropython-stubber\snippets\common\validate_gc.py
  C:\develop\MyPython\micropython-stubber\snippets\common\validate_gc.py:7:16 - error: "mem_free" is not a known member of module (reportGeneralTypeIssues)
  C:\develop\MyPython\micropython-stubber\snippets\common\validate_gc.py:13:14 - error: "mem_free" is not a known member of module (reportGeneralTypeIssues)
  C:\develop\MyPython\micropython-stubber\snippets\common\validate_gc.py:20:4 - error: "threshold" is not a known member of module (reportGeneralTypeIssues)
  C:\develop\MyPython\micropython-stubber\snippets\common\validate_gc.py:20:17 - error: "mem_free" is not a known member of module (reportGeneralTypeIssues)
  C:\develop\MyPython\micropython-stubber\snippets\common\validate_gc.py:20:38 - error: "mem_alloc" is not a known member of module (reportGeneralTypeIssues)
```

**Comment by Josverl** [comment]
fixed 19/11/222
Published  micropython-esp32-stubs              1.19.1.post7
Published  micropython-esp32-um_tinypico-stubs  1.19.1.post4
Published  micropython-esp8266-stubs            1.19.1.post6
....

**Comment by Josverl** [comment]
add below to stdlib.sys.pyi to resolve these
```
# micropython specific functions
def atexit(func:Callable) -> NoReturn: 
    """\
    Register func to be called upon termination. func must be a callable that takes no arguments, 
    or None to disable the call. The atexit function will return the previous value set by this function, 
    which is initially None.
    """
    ...

def print_exception(exc, file=sys.stdout, /):
    """Print exception with a traceback to a file-like object file (or sys.stdout by default)."""
    ...
```
